mantidproject/mantid
# Remove qt3 from project view

Qt3 classes have been removed from the Project Explorer (Folder and related classes).

**To test:**
- Open the Project Explorer. Check that (almost) all options work.
- Save and load projects.

Fixes #15315

 "Does not need to be in the release notes."

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
